sr_unix/gtm_init_env.c

```diff
@@ -14,12 +14,14 @@
 #include "stack_frame.h"
 #include "startup.h"
 #include "gtm_startup.h"
+#include "error.h"
 
 GBLREF	stack_frame	*frame_pointer;
 
 void gtm_init_env(rhdtyp *base_addr, unsigned char *transfer_addr)
 {
 	assert(CURRENT_RHEAD_ADR(base_addr) == base_addr);
+	ESTABLISH(mdb_condition_handler);
```

Comment:
I don't see any way to trigger an error inside gtm_init_env.c. The only error possibilities are STACKCRIT and STACKOFLOW. And because this function is invoked only when the process starts, the M stack is empty at that time AND the M-stack size is a minimum of 25KiB, there is no way of getting a STACKCRIT error for the one or two frames that we allocate at startup. So not sure why this change was necessary. Maybe it was done as part of general cleanup. I don't see any way having the ESTABLISH/REVERT makes any difference because of this. Given this, would you like removing this or keep it as is? My inclination is to leave it as is.

Reply:
Posting @estess response in the right thread.
------
I would like to remove it as I think it is dangerous as it sits in case
someone looks it over and figures it can deal with a newly added error
or something. I'm really curious as to the origin of that addition and
what it allegedly solved. Oh well.

Steve
-----------

Reply:
The ESTABLISH/REVERT of mdb_condition_handler in gtm_init_env.c is now removed.